src/page-modules/contact/travel-guarantee/index.tsx

```diff
@@ -0,0 +1,449 @@
+import FormSelector from './form-selector';
+import { SectionCard } from '../components/section-card';
+import { ComponentText, PageText, useTranslation } from '@atb/translations';
+import { useMachine } from '@xstate/react';
+import { fetchMachine } from './travelGuaranteeFormMachine';
+import { Input } from '../components/input';
+import { TransportModeType } from '@atb-as/config-specs';
+import { useLines } from '../lines/use-lines';
+import { FormEventHandler, useState } from 'react';
+import style from '../contact.module.css';
+import { Button } from '@atb/components/button';
+import Select from '../components/input/select';
+import { Line } from '..';
+import { Checkbox } from '../components/input/checkbox';
+import ErrorMessage from '../components/input/error-message';
+
+export const RefundForm = () => {
+  const { t } = useTranslation();
+  const { getLinesByMode, getQuaysByLine } = useLines();
+  const [state, send] = useMachine(fetchMachine);
+
+  // Local state to force re-render to display errors.
+  const [forsceRerender, setForsceRerender] = useState(false);
```

Comment:
```suggestion
  const [forceRerender, setForceRerender] = useState(false);
```

I'll take a look at the use-case for this later, but won't block merge for now.